x-pack/plugins/maps/public/components/metrics_editor/metric_editor.tsx

```diff
@@ -85,6 +99,20 @@ export function MetricEditor({
       field: fieldName,
     });
   };
+
+  const onPercentileChange = (e: ChangeEvent<HTMLInputElement> | MouseEvent<HTMLButtonElement>) => {
```

Comment:
Since the input is now a number input, you need to properly handle empty string.

<img width="460" alt="Screen Shot 2020-12-10 at 4 47 24 PM" src="https://user-images.githubusercontent.com/373691/101843629-dcca7e00-3b07-11eb-88e6-fe8ac10d49f9.png">